plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024 Marzieh Raoufnezhad <raoufnezhad at gmail.com>
+# Copyright (c) 2024 Maryam Mayabi <mayabi.ahm at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_info
+
+short_description: Proxmox Schedule Backup Info  
+
+description: The `proxmox_backup_info` module displays information such as backup times, VM name, VM ID, mode, backup type, and backup schedule using the Proxmox Server API.
+
+author: 
+    - Marzieh Raoufnezhad
+    - Maryam Mayabi
+
+options:
+      api_user:
+        description: The user must have access to the proxmox server 
+        required: true
+      api_password:
+        description: The password for the proxmox user 
+        required: true
+      api_host:
+        description: Poroxmox server domain name or ip address
+        required: true
+      vm_name:
+        description: Proxmox vm name
+        required: false
+      vm_id:
+        description: Proxmox vm id
+        required: false
+      backup_section:
+        description: proxmox_backup_section
+        required: false
+        defulat: false
+"""
+
+EXAMPLES = """
+---
+- name: Print all backup information by vmid and vm name
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+
+---
+- name: Print proxmox backup information for a specific vm based on its name
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      vm_name: 'mailsrv'
+
+---
+- name: Print proxmox backup information for a specific vm based on its vmid
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      vm_id: '150'
+
+---
+- name: Print proxmox all backup job information 
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
+      backup_section: 'true'
+"""
+
+RETURN = """
+---
+result:
+    description: The return value provided by proxmox_backup_info.
+    type: dict
```

Comment:
That description does not mean much. Please be either more specific or add notes to the documentation block referring to Proxmox API, and/or the return values provided by proxmoxer.

If the result value is not too big nor complex, please consider declaring `contains` block with a more finely grained level of details.